skimage/measure/_regionprops.py

```diff
@@ -569,6 +572,11 @@ def intensity_min(self):
         vals = self.image_intensity[self.image]
         return np.min(vals, axis=0).astype(np.float64, copy=False)
 
+    @property
+    def intensity_std(self):
+        vals = self.image_intensity[self.image]
+        return np.std(vals)
```

Comment:
The tests currently fail because this property doesn't account for the multichannel case.

```suggestion
        return np.std(vals, axis=0)
```
This should make it work.

Reply:
I think we should add a test for that.

Reply:
There already is: https://github.com/scikit-image/scikit-image/blob/c2725eaba2d3d84db329c6f35abc0b035efa3063/skimage/measure/tests/test_regionprops.py#L1466

That was the one that failed previously and is now passing with https://github.com/scikit-image/scikit-image/pull/6712/commits/3961a7eb078ed0ac9d8bb8bfbfce5185e901727f. Note that the other `intensity_*` properties also only rely on the above test only (seems to me that way).

Personally I'm content with the current state but I you want I can add explicit ones.